server/src/main/java/org/apache/gravitino/server/web/filter/GravitinoInterceptionService.java

```diff
@@ -234,10 +234,8 @@ private Map<Entity.EntityType, NameIdentifier> extractNameIdentifierFromParamete
                     NameIdentifierUtil.ofFileset(metalake, catalog, schema, fileset));
                 break;
               case MODEL:
-                String model = entities.get(Entity.EntityType.MODEL);
                 nameIdentifierMap.put(
-                    Entity.EntityType.MODEL,
-                    NameIdentifierUtil.ofModel(metadata, catalog, schema, model));
```

Comment:
no need to change the identifier, that'd lead to issues. You just need to replace `metadata` with `metalake` in the `ofModel` parameters in the original code.